### DISCUSSION TITLE
Update gtk-3.0 theme thumbnail.

### DISCUSSION THREAD

**Discussion by Initu-Castilhos** [ubuntu/yaru#3462] [General]
Sorry to bother you again but I figured out how to generate thumbnail for gtk-3.0 themes, there is a text file explaining how to generate thumbnail for themes located at:/usr/share/cinnamon/gtk-3.0.

![thumbnail](https://user-images.githubusercontent.com/59381215/157089122-4b55f9d5-d8fb-4e2a-8c04-af997ea13efd.png)
![thumbnail-dark](https://user-images.githubusercontent.com/59381215/157089157-162ecbae-06fc-47dc-93b4-f5dbf29080ac.png)
Here are the images.

The tutorial is this:

 HOW TO MAKE A GTK THEME THUMBNAIL?
 ----------------------------------

 - Install ImageMagick ("apt install imagemagick")
 - Open a terminal and run "cinnamon-preview-gtk-theme mytheme" (where mytheme is the name of the Theme you want to thumbnail)
 - Press Alt+Print-Screen to take a screenshot of the window
 - Go to your Pictures directory and move the screenshot to ~/screenshot.png
 - Open a terminal and type:

	cd
	convert screenshot.png -gravity north-west -extent 120x35 thumbnail.png

 WHICH THUMBNAIL IS USED BY CINNAMON TO REPRESENT A THEME?
 ---------------------------------------------------------

 - Cinnamon first tries to find a gtk-3.0/thumbnail.png file in the theme directory
 - If it doesn't find any, it looks in /usr/share/cinnamon/thumbnails/gtk-3.0/

Again I apologize but I thought it was important to pass this information on to future compilations.

**Comment by Jupi007** [comment]
Thanks a lot for this @Initu-Castilhos :+1: 
I will be useful for future updates.

**Comment by Initu-Castilhos** [comment]
I've made an update to the image cropping if you're interested in creating a P.R. just download the images.

![thumbnail](https://user-images.githubusercontent.com/59381215/157277104-92075cdc-9c04-4307-93d3-857bd332dda7.png)
![thumbnail-dark](https://user-images.githubusercontent.com/59381215/157277163-c4bdcf0a-fdbb-4ed1-a2a0-ca75601d49a8.png)

**Reply by Jupi007** [reply]
I could do it, but what about doing this by yourself.
So you being attributed for your work :)

**Reply by Initu-Castilhos** [reply]
I can't create a PR.

**Reply by Jupi007** [reply]
No problem, I'll do it so :)

**Reply by Initu-Castilhos** [reply]
Thanks!
